lib/Dialects/LinalgTransform/Transforms/TransformInterpreter.cpp

```diff
@@ -546,26 +668,73 @@ findTransformTarget(OpTy transformOp, ModuleOp module,
   return applyPatternsOnce(module, std::move(patterns));
 }
 
+template <typename OpTy>
+static std::enable_if_t<!llvm::is_detected<has_target_matcher_t, OpTy>::value,
+                FailureOr<SmallVector<Operation *>>>
+matchUsingPDL(OpTy transformOp, ModuleOp module) {
+  assert(false && "matching with PDL on an unsupported op");
+}
+
+/// Returns the operations that `transformOp` should be run on. The
```

Comment:
This sfinae genuflexion would go away once #218 lands right ?

Reply:
Most of it, not all.